Added resources for Figma

cburton4 opened the pull request:
## Description
Added resources and information to make it easier for folks to use Figma as a design tool.

cburton4 committed:
Added resources for Figma

Added resources and information to make it easier for folks to use Figma as a design tool.

kjellr commented on docs/designers-developers/designers/design-resources.md:
```diff
@@ -1,6 +1,57 @@
+[Figma](https://www.figma.com/) includes a library of [Gutenberg UI components](https://www.figma.com/file/ZtN5xslEVYgzU7Dd5CxgGZwq/WordPress-Components?node-id=491%3A3280) found in the `@wordpress/components` npm package, or “WordPress Components”. They are stable, fully supported, up to date, and ready for use in designs and prototypes. `@wordpress/components` is an npm package that is publishe ...
+
+### How to get access
+To join the WordPress.org Figma team, ping in #design on chat.wordpress.org and let us know whether you need view-only or edit access. A Figma admin will contact you for your email address to set up your account.
```
There's some copy here that we may be able to borrow/duplicate: https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#how-can-designers-contribute

cburton4 replied:
Awesome, will take a look and add 👍 Thanks!

cburton4 replied:
Updated.

kjellr commented on docs/designers-developers/designers/design-resources.md:
```diff
@@ -1,6 +1,57 @@
+
+If you prefer to use [Sketch](https://www.sketch.com/), the [SketchPress](https://github.com/10up/SketchPress) project includes a library of Gutenberg design components helpful for designing and prototyping:
 
 [Download Sketch mockups & patterns files](https://github.com/10up/SketchPress)
```
It doesn't look like these have been updated since February. Does it make sense to keep these included here?

cburton4 replied:
That's a good point. Might be best to remove this option in favor for the updated files.

cburton4 replied:
Removed.

kjellr commented on docs/designers-developers/designers/design-resources.md:
```diff
@@ -1,6 +1,57 @@
-The [SketchPress](https://github.com/10up/SketchPress) project includes a library of Gutenberg design components helpful for designing and prototyping:
+## Figma
+
+[Figma](https://www.figma.com/) includes a library of [Gutenberg UI components](https://www.figma.com/file/ZtN5xslEVYgzU7Dd5CxgGZwq/WordPress-Components?node-id=491%3A3280) found in the `@wordpress/components` npm package, or “WordPress Components”. They are stable, fully supported, up to date, and ready for use in designs and prototypes. `@wordpress/components` is an npm package that is publishe ...
```
> They are stable, fully supported, up to date, and ready for use in designs and prototypes.

I can't even begin to explain how happy this line makes me. 🙌

cburton4 committed:
Removed content and copy updates

Removed the section that references Sketch resources as they are out of date. I also used phrasing from other documentation to make updates to the copy.

kjellr commented on docs/designers-developers/designers/design-resources.md:
```diff
@@ -1,11 +1,51 @@
+
+**How to refine or contribute to the WordPress components React library _(Coming soon)_**
+
+WordPress components in Figma mirror the live React components. Documentation for how to refine or contribute to WordPress components in React coming soon.
```
```suggestion
WordPress components in Figma mirror the live React components. Documentation for how to refine or contribute to WordPress components in React is coming soon.
```

kjellr reviewed (approved):
Left just one tiny copy suggestion. Other than that, I think this is ready to ship! Thanks, @cburton4!

cburton4 committed:
Update docs/designers-developers/designers/design-resources.md

Co-Authored-By: Kjell Reigstad <kjell.reigstad@automattic.com>
